# Исследование надежности заемщиков


## Цель исследования: 
Исследовать предоставленные банком данные и ответить на вопросы о зависимости между различныи факторами (количество детей, семейное положение, уровень дохода, цели кредита) и возвратом кредита в срок.

## Описание проекта:
Заказчик — кредитный отдел банка. Нужно разобраться, влияет ли семейное положение и количество детей клиента на факт погашения кредита в срок. Результаты исследования будут учтены при построении модели кредитного скоринга — специальной системы, которая оценивает способность потенциального заёмщика вернуть кредит банку.

## Описание данных: 
Таблица с данными от банка содержит в себе статистику о платёжеспособности клиентов. 
- children — количество детей в семье 
- days_employed — общий трудовой стаж в днях 
- dob_years — возраст клиента в годах 
- education — уровень образования клиента 
- education_id — идентификатор уровня образования 
- family_status — семейное положение 
- family_status_id — идентификатор семейного положения 
- gender — пол клиента 
- income_type — тип занятости 
- debt — имел ли задолженность по возврату кредитов 
- total_income — ежемесячный доход purpose — цель получения креди

## План работы:
- Ознакомиться с предоставленными данными
- Провести предобработку данных (избавиться от пропусков, дубликатов и ошибок в данных)
- Провести исследовательский анализ данных (категоризировать данные, построить сводные таблицы)
- Сформулировать выводы о проделанной работе

Во второй части проекта вы выполните шаги 3 и 4. Их вручную проверит ревьюер.
Чтобы вам не пришлось писать код заново для шагов 1 и 2, мы добавили авторские решения в ячейки с кодом. 



## Откройте таблицу и изучите общую информацию о данных

**Задание 1. Импортируйте библиотеку pandas. Считайте данные из csv-файла в датафрейм и сохраните в переменную `data`. Путь к файлу:**

`/datasets/data.csv`

In [21]:
import pandas as pd

In [22]:
try:
    data = pd.read_csv('/datasets/data.csv')
except:
    data = pd.read_csv('https://code.s3.yandex.net/datasets/data.csv')

**Задание 2. Выведите первые 20 строчек датафрейма `data` на экран.**

In [23]:
data.head(20)

children  days_employed  dob_years            education  education_id  \
0          1   -8437.673028         42               высшее             0   
1          1   -4024.803754         36              среднее             1   
2          0   -5623.422610         33              Среднее             1   
3          3   -4124.747207         32              среднее             1   
4          0  340266.072047         53              среднее             1   
5          0    -926.185831         27               высшее             0   
6          0   -2879.202052         43               высшее             0   
7          0    -152.779569         50              СРЕДНЕЕ             1   
8          2   -6929.865299         35               ВЫСШЕЕ             0   
9          0   -2188.756445         41              среднее             1   
10         2   -4171.483647         36               высшее             0   
11         0    -792.701887         40              среднее             1   
12         0            NaN         65              среднее             1   
13         0   -1846.641941         54  неоконченное высшее             2   
14         0   -1844.956182         56               высшее             0   
15         1    -972.364419         26              среднее             1   
16         0   -1719.934226         35              среднее             1   
17         0   -2369.999720         33               высшее             0   
18         0  400281.136913         53              среднее             1   
19         0  -10038.818549         48              СРЕДНЕЕ             1   

       family_status  family_status_id gender income_type  debt  \
0    женат / замужем                 0      F   сотрудник     0   
1    женат / замужем                 0      F   сотрудник     0   
2    женат / замужем                 0      M   сотрудник     0   
3    женат / замужем                 0      M   сотрудник     0   
4   гражданский брак                 1      F   пенсионер     0   
5   гражданский брак                 1      M   компаньон     0   
6    женат / замужем                 0      F   компаньон     0   
7    женат / замужем                 0      M   сотрудник     0   
8   гражданский брак                 1      F   сотрудник     0   
9    женат / замужем                 0      M   сотрудник     0   
10   женат / замужем                 0      M   компаньон     0   
11   женат / замужем                 0      F   сотрудник     0   
12  гражданский брак                 1      M   пенсионер     0   
13   женат / замужем                 0      F   сотрудник     0   
14  гражданский брак                 1      F   компаньон     1   
15   женат / замужем                 0      F   сотрудник     0   
16   женат / замужем                 0      F   сотрудник     0   
17  гражданский брак                 1      M   сотрудник     0   
18    вдовец / вдова                 2      F   пенсионер     0   
19         в разводе                 3      F   сотрудник     0   

     total_income                                 purpose  
0   253875.639453                           покупка жилья  
1   112080.014102                 приобретение автомобиля  
2   145885.952297                           покупка жилья  
3   267628.550329              дополнительное образование  
4   158616.077870                         сыграть свадьбу  
5   255763.565419                           покупка жилья  
6   240525.971920                       операции с жильем  
7   135823.934197                             образование  
8    95856.832424                   на проведение свадьбы  
9   144425.938277                 покупка жилья для семьи  
10  113943.491460                    покупка недвижимости  
11   77069.234271       покупка коммерческой недвижимости  
12            NaN                         сыграть свадьбу  
13  130458.228857                 приобретение автомобиля  
14  165127.911772              покупка жилой недвижимости  
15  116820.904450 

**Задание 3. Выведите основную информацию о датафрейме с помощью метода `info()`.**

In [24]:
data.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 21525 entries, 0 to 21524
Data columns (total 12 columns):
 #   Column            Non-Null Count  Dtype  
---  ------            --------------  -----  
 0   children          21525 non-null  int64  
 1   days_employed     19351 non-null  float64
 2   dob_years         21525 non-null  int64  
 3   education         21525 non-null  object 
 4   education_id      21525 non-null  int64  
 5   family_status     21525 non-null  object 
 6   family_status_id  21525 non-null  int64  
 7   gender            21525 non-null  object 
 8   income_type       21525 non-null  object 
 9   debt              21525 non-null  int64  
 10  total_income      19351 non-null  float64
 11  purpose           21525 non-null  object 
dtypes: float64(2), int64(5), object(5)
memory usage: 2.0+ MB


## Предобработка данных

### Удаление пропусков

**Задание 4. Выведите количество пропущенных значений для каждого столбца. Используйте комбинацию двух методов.**

In [25]:
data.isna().sum()

children               0
days_employed       2174
dob_years              0
education              0
education_id           0
family_status          0
family_status_id       0
gender                 0
income_type            0
debt                   0
total_income        2174
purpose                0
dtype: int64

**Задание 5. В двух столбцах есть пропущенные значения. Один из них — `days_employed`. Пропуски в этом столбце вы обработаете на следующем этапе. Другой столбец с пропущенными значениями — `total_income` — хранит данные о доходах. На сумму дохода сильнее всего влияет тип занятости, поэтому заполнить пропуски в этом столбце нужно медианным значением по каждому типу из столбца `income_type`. Например, у человека с типом занятости `сотрудник` пропуск в столбце `total_income` должен быть заполнен медианным доходом среди всех записей с тем же типом.**

In [26]:
for t in data['income_type'].unique():
    data.loc[(data['income_type'] == t) & (data['total_income'].isna()), 'total_income'] = \
    data.loc[(data['income_type'] == t), 'total_income'].median()

### Обработка аномальных значений

**Задание 6. В данных могут встречаться артефакты (аномалии) — значения, которые не отражают действительность и появились по какой-то ошибке. таким артефактом будет отрицательное количество дней трудового стажа в столбце `days_employed`. Для реальных данных это нормально. Обработайте значения в этом столбце: замените все отрицательные значения положительными с помощью метода `abs()`.**

In [27]:
data['days_employed'] = data['days_employed'].abs()

**Задание 7. Для каждого типа занятости выведите медианное значение трудового стажа `days_employed` в днях.**

In [28]:
data.groupby('income_type')['days_employed'].agg('median')

income_type
безработный        366413.652744
в декрете            3296.759962
госслужащий          2689.368353
компаньон            1547.382223
пенсионер          365213.306266
предприниматель       520.848083
сотрудник            1574.202821
студент               578.751554
Name: days_employed, dtype: float64

У двух типов (безработные и пенсионеры) получатся аномально большие значения. Исправить такие значения сложно, поэтому оставьте их как есть.

**Задание 8. Выведите перечень уникальных значений столбца `children`.**

In [29]:
data['children'].unique()

array([ 1,  0,  3,  2, -1,  4, 20,  5])

**Задание 9. В столбце `children` есть два аномальных значения. Удалите строки, в которых встречаются такие аномальные значения из датафрейма `data`.**

In [30]:
data = data[(data['children'] != -1) & (data['children'] != 20)]

**Задание 10. Ещё раз выведите перечень уникальных значений столбца `children`, чтобы убедиться, что артефакты удалены.**

In [31]:
data['children'].unique()

array([1, 0, 3, 2, 4, 5])

### Удаление пропусков (продолжение)

**Задание 11. Заполните пропуски в столбце `days_employed` медианными значениями по каждого типа занятости `income_type`.**

In [32]:
for t in data['income_type'].unique():
    data.loc[(data['income_type'] == t) & (data['days_employed'].isna()), 'days_employed'] = \
    data.loc[(data['income_type'] == t), 'days_employed'].median()

**Задание 12. Убедитесь, что все пропуски заполнены. Проверьте себя и ещё раз выведите количество пропущенных значений для каждого столбца с помощью двух методов.**

In [33]:
data.isna().sum()

children            0
days_employed       0
dob_years           0
education           0
education_id        0
family_status       0
family_status_id    0
gender              0
income_type         0
debt                0
total_income        0
purpose             0
dtype: int64

### Изменение типов данных

**Задание 13. Замените вещественный тип данных в столбце `total_income` на целочисленный с помощью метода `astype()`.**

In [34]:
data['total_income'] = data['total_income'].astype(int)

### Обработка дубликатов

**Задание 14. Обработайте неявные дубликаты в столбце `education`. В этом столбце есть одни и те же значения, но записанные по-разному: с использованием заглавных и строчных букв. Приведите их к нижнему регистру.**

In [35]:
data['education'] = data['education'].str.lower()

**Задание 15. Выведите на экран количество строк-дубликатов в данных. Если такие строки присутствуют, удалите их.**

In [36]:
data.duplicated().sum()

71

In [37]:
data = data.drop_duplicates()

### Категоризация данных

**Задание 16. На основании диапазонов, указанных ниже, создайте в датафрейме `data` столбец `total_income_category` с категориями:**

- 0–30000 — `'E'`;
- 30001–50000 — `'D'`;
- 50001–200000 — `'C'`;
- 200001–1000000 — `'B'`;
- 1000001 и выше — `'A'`.


**Например, кредитополучателю с доходом 25000 нужно назначить категорию `'E'`, а клиенту, получающему 235000, — `'B'`. Используйте собственную функцию с именем `categorize_income()` и метод `apply()`.**

In [38]:
def categorize_income(income):
    try:
        if 0 <= income <= 30000:
            return 'E'
        elif 30001 <= income <= 50000:
            return 'D'
        elif 50001 <= income <= 200000:
            return 'C'
        elif 200001 <= income <= 1000000:
            return 'B'
        elif income >= 1000001:
            return 'A'
    except:
        pass

In [39]:
data['total_income_category'] = data['total_income'].apply(categorize_income)

**Задание 17. Выведите на экран перечень уникальных целей взятия кредита из столбца `purpose`.**

In [40]:
data['purpose'].unique()

array(['покупка жилья', 'приобретение автомобиля',
       'дополнительное образование', 'сыграть свадьбу',
       'операции с жильем', 'образование', 'на проведение свадьбы',
       'покупка жилья для семьи', 'покупка недвижимости',
       'покупка коммерческой недвижимости', 'покупка жилой недвижимости',
       'строительство собственной недвижимости', 'недвижимость',
       'строительство недвижимости', 'на покупку подержанного автомобиля',
       'на покупку своего автомобиля',
       'операции с коммерческой недвижимостью',
       'строительство жилой недвижимости', 'жилье',
       'операции со своей недвижимостью', 'автомобили',
       'заняться образованием', 'сделка с подержанным автомобилем',
       'получение образования', 'автомобиль', 'свадьба',
       'получение дополнительного образования', 'покупка своего жилья',
       'операции с недвижимостью', 'получение высшего образования',
       'свой автомобиль', 'сделка с автомобилем',
       'профильное образование', 'высшее об

**Задание 18. Создайте функцию, которая на основании данных из столбца `purpose` сформирует новый столбец `purpose_category`, в который войдут следующие категории:**

- `'операции с автомобилем'`,
- `'операции с недвижимостью'`,
- `'проведение свадьбы'`,
- `'получение образования'`.

**Например, если в столбце `purpose` находится подстрока `'на покупку автомобиля'`, то в столбце `purpose_category` должна появиться строка `'операции с автомобилем'`.**

**Используйте собственную функцию с именем `categorize_purpose()` и метод `apply()`. Изучите данные в столбце `purpose` и определите, какие подстроки помогут вам правильно определить категорию.**

In [41]:
def categorize_purpose(row):
    try:
        if 'автом' in row:
            return 'операции с автомобилем'
        elif 'жил' in row or 'недвиж' in row:
            return 'операции с недвижимостью'
        elif 'свад' in row:
            return 'проведение свадьбы'
        elif 'образов' in row:
            return 'получение образования'
    except:
        return 'нет категории'

In [42]:
data['purpose_category'] = data['purpose'].apply(categorize_purpose)

In [43]:
data.info()

<class 'pandas.core.frame.DataFrame'>
Int64Index: 21331 entries, 0 to 21524
Data columns (total 14 columns):
 #   Column                 Non-Null Count  Dtype  
---  ------                 --------------  -----  
 0   children               21331 non-null  int64  
 1   days_employed          21331 non-null  float64
 2   dob_years              21331 non-null  int64  
 3   education              21331 non-null  object 
 4   education_id           21331 non-null  int64  
 5   family_status          21331 non-null  object 
 6   family_status_id       21331 non-null  int64  
 7   gender                 21331 non-null  object 
 8   income_type            21331 non-null  object 
 9   debt                   21331 non-null  int64  
 10  total_income           21331 non-null  int64  
 11  purpose                21331 non-null  object 
 12  total_income_category  21331 non-null  object 
 13  purpose_category       21331 non-null  object 
dtypes: float64(1), int64(6), object(7)
memory usage: 2.4+ 

### Шаг 3. Исследуйте данные и ответьте на вопросы

#### 3.1 Есть ли зависимость между количеством детей и возвратом кредита в срок?

Чтобы ответить на поставленный вопрос, необходимо вычислить доли семей (в каждой категории), которые имели задолженности по возврату кредитов.
Затем сравнить и сделать выводы на осонве полученных данных.

In [44]:
#Распределение семей по количеству детей
data['children'].value_counts()

0    14091
1     4808
2     2052
3      330
4       41
5        9
Name: children, dtype: int64

Опираясь на представленную выше информацию, делаем вывод, что в предоставленной таблице недостаточно данных о семьях в которых 3, 4, и 5 детей. Эти выборки считаем нерепрезентативными, в анализе и выводах учитывать не будем
Категоризируем данные из столбца 'children', создадим собственную функцию categorized_children()

In [45]:
def categorized_children(row):
    if row == 0:
        return 'без детей'
    if row == 1:
        return '1 ребенок'
    if row == 2:
        return '2 детей'
    if row >= 3:
        return 'многодетные'


In [46]:
#Используем функцию categorized_children()и метод apply() для создания нового столбца data['children_category']:
data['children_category'] = data['children'].apply(categorized_children)

#Для самопроверки выведем первые 10 строк таблицы data при помощи метода head():
data.head(10)



children  days_employed  dob_years education  education_id  \
0         1    8437.673028         42    высшее             0   
1         1    4024.803754         36   среднее             1   
2         0    5623.422610         33   среднее             1   
3         3    4124.747207         32   среднее             1   
4         0  340266.072047         53   среднее             1   
5         0     926.185831         27    высшее             0   
6         0    2879.202052         43    высшее             0   
7         0     152.779569         50   среднее             1   
8         2    6929.865299         35    высшее             0   
9         0    2188.756445         41   среднее             1   

      family_status  family_status_id gender income_type  debt  total_income  \
0   женат / замужем                 0      F   сотрудник     0        253875   
1   женат / замужем                 0      F   сотрудник     0        112080   
2   женат / замужем                 0      M   сотрудник     0        145885   
3   женат / замужем                 0      M   сотрудник     0        267628   
4  гражданский брак                 1      F   пенсионер     0        158616   
5  гражданский брак                 1      M   компаньон     0        255763   
6   женат / замужем                 0      F   компаньон     0        240525   
7   женат / замужем                 0      M   сотрудник     0        135823   
8  гражданский брак                 1      F   сотрудник     0         95856   
9   женат / замужем                 0      M   сотрудник     0        144425   

                      purpose total_income_category          purpose_category  \
0               покупка жилья                     B  операции с недвижимостью   
1     приобретение автомобиля                     C    операции с автомобилем   
2               покупка жилья                     C  операции с недвижимостью   
3  дополнительное образование                     B     получение образования   
4             сыграть свадьбу                     C        проведение свадьбы   
5               покупка жилья                     B  операции с недвижимостью   
6           операции с жильем                     B  операции с недвижимостью   
7                 образование                     C     получение образования   
8       на проведение свадьбы                     C        проведение свадьбы   
9     покупка жилья для семьи                     C  операции с недвижимостью   

  children_category  
0         1 ребенок  
1         1 ребенок  
2         без детей  
3       многодетные  
4         без детей  
5         без детей  
6         без детей  
7         без детей  
8           2 детей  
9         без детей

In [47]:
#При помощи метода pivot_table() создаем сводную таблицу
children_pivot = data.pivot_table(index = ['children_category'], \
                                columns = ['debt'], \
                                values = 'purpose', aggfunc='count')
#Расчитываем долю невозвращенных кредитов в каждой из категорий (число должников/общее число семей)
children_pivot['ratio_children'] = (children_pivot[1] / (children_pivot[1]+children_pivot[0]))
#Преобразуем результат из долей в проценты
children_pivot['ratio_children'] = children_pivot['ratio_children'].map('{:.1%}'.format)
children_pivot

debt                   0     1 ratio_children
children_category                            
1 ребенок           4364   444           9.2%
2 детей             1858   194           9.5%
без детей          13028  1063           7.5%
многодетные          349    31           8.2%

.**Вывод:** 
В категории семей "без детей" доля задолженностей меньше, чем в остальных категориях.
Вероятно это связано с отсутствием расходов на их обеспечение.
Однако следующими в данном "рейтинге" оказались многодетные семьи. Это может быть связанно с тем, что государство оказывает различную помощь таким семьям, в том числе, государственные льготы, соцподдержка,кредиты на льготных условиях. В виду недостатка данных в итоговый вывод категорию "многодетные" не включаем.
В семьях с одним и двумя детьми разница незначительна. Такие семьи с большей вероятностью окажутся должниками.

#### 3.2 Есть ли зависимость между семейным положением и возвратом кредита в срок?

In [48]:
# Для начала, ознакомимся с имеющимися данными
# Выведем на экран все уникальные значения стобца 'family_status':
data['family_status'].value_counts()

женат / замужем          12261
гражданский брак          4134
Не женат / не замужем     2796
в разводе                 1189
вдовец / вдова             951
Name: family_status, dtype: int64

Выведем на экран сводную таблицу, содержащую с добросовестных плательщиков и должников 
по категориям семейного положения.
Затем расчитаем отношение должников к общему числу(в процентном соотношении). 
На основе полученных расчетов сформулируем выводы.

In [49]:
#Сводная таблица
pivot_family_status = data.pivot_table(index = ['family_status'], \
                                columns = ['debt'], \
                                values = 'purpose', aggfunc='count')
#Результат, выраженный десятичной дробью
pivot_family_status['ratio_family'] = pivot_family_status[1]/(pivot_family_status[0]+pivot_family_status[1])
#Переводим в более удобный для работы формат:одна цифра после запятой, результат в процентах:
pivot_family_status['ratio_family'] = pivot_family_status['ratio_family'].map('{:.1%}'.format)
pivot_family_status

debt                       0    1 ratio_family
family_status                                 
Не женат / не замужем   2523  273         9.8%
в разводе               1105   84         7.1%
вдовец / вдова           888   63         6.6%
гражданский брак        3749  385         9.3%
женат / замужем        11334  927         7.6%

**Вывод:** 
Наибольший процент возникновения задолженности у категорий "Не женат / не замужем" и "гражданский брак".
Более исправно платят кредиты люди из категорий "вдовец / вдова", "в разводе", "женат / замужем"


#### 3.3 Есть ли зависимость между уровнем дохода и возвратом кредита в срок?

В данном проекте(В задании 16) мы уже категоризировали данные из столбца ['total_income'].
Далее будем проводить исследование с уже имеющимися категориями (столбец ['total_income_category]:
0–30000 — 'E';
30001–50000 — 'D';
50001–200000 — 'C';
200001–1000000 — 'B';
1000001 и выше — 'A'.


In [50]:
#Создаем сводную таблицу
pivot_total_income_category = data.pivot_table(index = ['total_income_category'], \
                                columns = ['debt'], \
                                values = 'purpose', aggfunc='count')
pivot_total_income_category['ratio_income'] = pivot_total_income_category[1]/(pivot_total_income_category[1]+pivot_total_income_category[0])
pivot_total_income_category['ratio_income'] = pivot_total_income_category['ratio_income'].map('{:.1%}'.format)
pivot_total_income_category

debt                       0     1 ratio_income
total_income_category                          
A                         23     2         8.0%
B                       4660   354         7.1%
C                      14568  1353         8.5%
D                        328    21         6.0%
E                         20     2         9.1%

Категории А, Е и D не берем в расчет при анализе, т.к. они являются нерепрезентативными из-за недостаточного 
количества данных.

**Вывод:** 
Сравниваем только группы В и С, т.к. данные выборки являются репрезентативными.
На основании полученных расчетов, у категории С (люди со средним доходом) наибольшая вероятность
возникновения задолженности, по сравнению с категорией В.

#### 3.4 Как разные цели кредита влияют на его возврат в срок?

Ранее, в задании 18, мы уже разбили многообразие целей кредита на 4 категории.
Выведем на экран из уникальные значения.

In [51]:
data['purpose_category'].unique()

array(['операции с недвижимостью', 'операции с автомобилем',
       'получение образования', 'проведение свадьбы'], dtype=object)

In [52]:
#Составим сводную таблицу:
pivot_purpose_category = data.pivot_table (index = ['purpose_category'], \
                                columns = ['debt'], \
                                values = 'purpose', aggfunc='count')
pivot_purpose_category['ratio_purpose'] = pivot_purpose_category[1]/(pivot_purpose_category[0]+pivot_purpose_category[1])
pivot_purpose_category['ratio_purpose'] = pivot_purpose_category['ratio_purpose'].map('{:.1%}'.format)
pivot_purpose_category

debt                         0    1 ratio_purpose
purpose_category                                 
операции с автомобилем    3879  400          9.3%
операции с недвижимостью  9971  780          7.3%
получение образования     3619  369          9.3%
проведение свадьбы        2130  183          7.9%

**Вывод:** 
Категории людей,которые берут кредит на обучение и операции с автомобилем, наиболее подвержены риску оказаться должниками перед банком.
Сделки с недвижимостью являются наиболее надежными для банка, т.к.клиенты более исправно платят по таким кредитам.

#### 3.5 Приведите возможные причины появления пропусков в исходных данных.

*Ответ:* 

Причинами появления пропусков в данных могут быть: для случайных пропуской - человеческий фактор; ошибки при вводе данных; технические ошибки при сохранении файла, конвертации в другие форматы; повреждение файла.
Бывает, что пропуски появляются не из-за чьей-то ошибки, а тоже несут в себе информацию. Наличие таких пропусков может быть объяснено данными из других столбцов таблицы.

#### 3.6 Объясните, почему заполнить пропуски медианным значением — лучшее решение для количественных переменных.

*Ответ:* 
Пропуски, заполненные медианными значениями, в некоторых случаях, являются наиболее конструктивным решением для замены отсутствующих данных. Медианные значения наиболее объективно отражают восполнение упущенной информации,т.к.являются типичными для данной выборки. Медиана устойчива к выбросам и аномальным значениям. Такое решение - один из наиболее простых и быстрых методов обработки пропусков(наряду со средним арифметическим. Это важно, когда данные и выводы из них требуются срочно.
Медианное значение является типичным для данной выборки,часто используется как альтернатива среднему значению, а также 

### Шаг 4: общий вывод.

1)Как правило, чем меньше детей, тем меньше уровень долговой нагрузки, и тем легче семья справится с выплатой займа. Так, разница между категориями "без детей"(7.5%) и с 1-2 детьми(9.2%-9.5%) составляет 1,8-2,0%\
2)В данных прослеживается следующая тенденция: "Люди, которые сотоят, или состояли ранее в официальном браке имеют меньше рисков стать должниками" Граждане, побывавшие в браке, в среднем, на 2-3% успешнее холостых справляются с выплатой задолженностей. Брак воспитывает в людях чувство обязательства и ответственности.
Так, овдовевшие люди лучше остальных групп справляются с выплатой кредитов (6.6%), далее, по возрастанию следуют разведенные(7.1%), женатые/замужние(7.6%). Хуже всех справляются с выплатой долга состоящие в гражданском браке (9.3%) и не состоящие в браке(9,8%)\
3)Вероятность возникновения задолженности выше в группе С(с уровнем заработка 50001–200000) составляет 8.5%, против 7,1% в группе В (с уровнем заработка 200001–1000000)
Следовательно, более платежеспособное население является более надежными клиентами, так как реже допускает возникновение задолженностей.\
4)Граждане наиболее ответственно относятся к возврату займов, по операциям с недвижимостью (7,3%)т.к они берутся на крупные суммы на длительный срок и под залог недвижимости. 
Вероятность оказаться должником,устраивая свадьбу небольшая, и составляет 7.9%. Часть кредита оплачивается после проведения мероприятия, за счет свадебных подарков.\
Более всего подвержены риску задолженности граждане, взявшие займы на операции с автомобилем и получение образования - 9.3%\

**Подведем итоги:**

Банк будет меньше рисковать, если прежде всего отдаст предпочтение в выдаче займов следующим категориям:
- Без детей (7.5%),
- вдовец/вдова (6.6%),
- с уровнем дохода 200001–1000000 - группа В (6.0%),
- на операции с недвижимостью (7,3%).